BarnzNhutt.cpp

```diff
@@ -235,17 +238,14 @@ void createFrame(char* image, double* hdImage, struct body* b, int step)
 
 void renderClear(char* image, double* hdImage)
 {
-	for (int i=0; i<WIDTH*HEIGHT*3; i++)
-	{
-	//	char* current = image + i;
-		image[i] = 0; //char(image[i]/1.2);
-		hdImage[i] = 0.0;
-	}
+	memset(image, 0, WIDTH*HEIGHT*3);
+	memset(hdImage, 0, WIDTH*HEIGHT*3*sizeof(double));
 }
 
 void renderBodies(struct body* b, double* hdImage)
 {
 	/// ORTHOGONAL PROJECTION
+	#pragma omp parallel
```

Comment:
This seems to be slower than the serial version- at least with the system I'm currently testing on